js/app.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+const ItemAll = [];
+function Item(data) {
+ this.url = data.image_url;
+ this.title = data.title;
+ this.description = data.description;
+ this.keyword = data.keyword;
+ this.horns = data.horns;
+ ItemAll.push(this);
+}
+const keywordArray = [];
+function renderAnyHandlebars(sourceId, data, target) {
```

Comment:
Did you write this yourself, find it, or work with a partner?  It's a cool helper function, but I want to make sure you really understand how it operates.  If you borrowed or collaborated, please cite your source!